presto-main/src/main/java/com/facebook/presto/security/AllowAllSystemAccessControl.java

```diff
@@ -187,4 +187,15 @@ public void checkCanGrantTablePrivilege(Identity identity, Privilege privilege,
     public void checkCanRevokeTablePrivilege(Identity identity, Privilege privilege, CatalogSchemaTableName table, PrestoPrincipal revokee, boolean grantOptionFor)
     {
     }
+
+    @Override
+    public void checkCanShowRoles(Identity identity, String catalogName)
+    {
+    }
+
+    @Override
+    public Set<String> filterRoles(Identity identity, String catalogName, Set<String> roles)
```

Comment:
This method was dropped in https://github.com/prestosql/presto/pull/90/commits/e2a2ff03d78734065ef0165225cebbb9dc3af89f